### ISSUE TITLE
dstat crashes when screen is too small and logging with --output

### ISSUE THREAD

**Issue by cmoxiv** [dagwieers/dstat#111]
##### SUMMARY
The system monitor tool crashes when logging via "--output file.csv" option if the screen is too small

##### ISSUE TYPE
 - Bug Report

##### DSTAT VERSION
Dstat 0.7.2

##### OS / ENVIRONMENT
Ubuntu 14

##### STEPS TO REPRODUCE
```
$ dstat --cpu -C 0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16 --output test.csv
```

##### EXPECTED RESULTS
Exporting the stats into file.csv while displaying what the tool can afford within the terminal estate.

##### ACTUAL RESULTS
The tool crashes and logging stops. 

```
Terminal width too small, trimming output.
Traceback (most recent call last):
  File "/usr/bin/dstat", line 2554, in <module>
    main()
  File "/usr/bin/dstat", line 2416, in main
    scheduler.run()
  File "/usr/lib/python2.7/sched.py", line 117, in run
    action(*argument)
  File "/usr/bin/dstat", line 2514, in perform
    oline = oline + o.showcsv() + o.showcsvend(totlist, vislist)
  File "/usr/bin/dstat", line 542, in showcsvend
    if self is not vislist[-1]:
IndexError: list index out of range
```

##### WORKAROUND
Directing the dstat output to /dev/null 
```
$ dstat --cpu -C 0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16 --output test.csv > /dev/null
```

**Comment by dagwieers** [comment]
Ok, this seems trivial to fix. Because the list of visible plugins is empty (no plugins have output that matches the screen-size), refering to the last item in that list will fail.

We should check if the list is not empty first. Amazing that this wasn't reported before ? Good catch.

**Comment by dagwieers** [comment]
Ah, this is already fixed in v0.7.3 !

As per the instructions in the issue-template, before reporting an issue please check if it still occurs with the latest release.
